piws/demo/import_toy_dataset.py

```diff
@@ -41,14 +40,20 @@
 demo_path = raw_input("\nEnter where are the demo data [default: /tmp/demo]: ")
 if not demo_path:
     demo_path = "/tmp/demo"
-login = raw_input("\nEnter the '{0}' login [default: anon]: ".format(
-    instance_name))
-if not login:
-    login = "anon"
-password = getpass.getpass("Enter the '{0}' password [default: anon]: ".format(
-    instance_name))
-if not password:
-    password = "anon"
+available_stores = ["None", "SQLGenObjectStore", "MassiveObjectStore"]
+menu = "\nAvailable importation methods: "
```

Comment:
Add special select insertion method paragraph

# Select the insertion methode type
...